.github/workflows/actions.yml

```diff
@@ -0,0 +1,44 @@
+name: Continuous Integration
+on:
+  push:
+    path-ignore:
+      - docs/README.md
+  pull_request:
+    branches:
+      - master
+    path-ignore:
+      - docs/README.md
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag mytest .
+    - run: docker run --env-file .env.test mytest
+  prod-check:
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
This is important, but also worth removing for at least one commit to check the later steps work - I can't see any example runs that make it obvious that the below has succeeded, and in fact your app (http://devopscourse1509.herokuapp.com/) is currently still hosting the dotnet app that we used in the M8 workshop!